Assets/Feature01/Scripts/Bullet/BlueBullet.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BlueBullet : BulletAbstractClasses
+{
+    Color BlueColorBullet = Color.blue;
```

Comment:
Wrong. Color property should be on Abstract class. It should just be called Color. Not Blue Color or red color. If you don't use generic name you won't be able to use generic codes.